From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@DingyuanHu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Blackmist commented:
Looks good, please merge.

Commit:
updated branding from Web Apps for Containers to App Service on Linux

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

Commit:
branding updates

Commit:
branding updates

Commit:
branding updates

HeidiSteen committed:
edits

Commit:
updating branding in breadcrumbs

Commit:
updated some web app wording

Commit:
updating faq branding

Commit:
adding .net branding changes

Commit:
updated ruby branding

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'heidist-master' of https://github.com/HeidiSteen/azure-docs-pr; branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
initial draft

HeidiSteen committed:
toc edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
toc edits

HeidiSteen committed:
facet art, div list, etc

HeidiSteen committed:
h2 fix

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
better art

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
filter edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
typos, cleanup

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
updated branding, new linux app service include

Commit:
updated branding, new linux app service include

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
prep for publish

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
broken links

HeidiSteen committed:
Bruce edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
facet cleanup

HeidiSteen committed:
security trim

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
latest edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

aborts:wq
:wq

HeidiSteen committed:
x

GitHubber17 committed:
edit pass: load-balancer-internal-overview

GitHubber17 committed:
edit pass: load-balancer-distribution-mode

GitHubber17 committed:
edit pass: load-balancer-multiple-ip

HeidiSteen committed:
edits

GitHubber17 committed:
edit pass: load-balancer-get-started-ilb-arm-ps + includes

GitHubber17 committed:
edit pass: load-balancer-standard-overview + include

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
cleanup

HeidiSteen committed:
edits to filter def and pipeline

HeidiSteen committed:
facets ready to go?

Commit:
fixed nonblocking issues from PR#25221

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
JL feedback plus did a git-pull on Revital's doc

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
broken links

HeidiSteen committed:
edits to analyzers

itechedit committed:
edit pass: active-directory-b2c-ui-customization-custom-dynamic

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-ui-customization-custom-dynamic

itechedit committed:
edit pass: active-directory-b2c-ui-customization-custom-dynamic

HeidiSteen committed:
Merge branch 'heidist-master' of https://github.com/HeidiSteen/azure-docs-pr; branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
TOC bug, spacing

itechedit committed:
Update active-directory-b2c-ui-customization-custom-dynamic.md

Commit:
updating images with proper branding

Commit:
updated file including new screenshots

haroldwongms committed:
Update TOC to include additional OpenShift documentation

haroldwongms committed:
OpenShift Origin Documentation

Inculde Origin specific instructions

haroldwongms committed:
OpenShift Container Platform Documentation

Include Container Platform specific instructions - both ARM Template and
Azure Marketplace.

haroldwongms committed:
OpenShift Pre-Requisites for Azure

Include Pre-requisites for Azure that applies to Origin and Container
Platform - Key Vault, Service Principal

haroldwongms committed:
OpenShift Post Deployment tasks

Include Azure AD integration for SSO, enbaling Metrics and Logging, and
configure OMS for monitoring

haroldwongms committed:
OpenShift Troubleshooting for Azure

Troubleshooting details for deployments into Azure

haroldwongms committed:
Update with new Overview Information

haroldwongms committed:
Spelling and grammar cleanup

haroldwongms committed:
Include screenshots

haroldwongms committed:
Correct link

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

GitHubber17 committed:
resolve merge conflicts

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
JL, MHKO, RB feedback

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
adding fiddler doc back in

HeidiSteen committed:
picked up the analyzer edits

HeidiSteen committed:
acro edits

haroldwongms committed:
Remove jpgs

haroldwongms committed:
Add PNG with grey border

haroldwongms committed:
Changes per review feedback

haroldwongms committed:
Correct name of file

haroldwongms committed:
Remove reference to tutorial

haroldwongms committed:
updated image

ggailey777 committed:
Remove the storage gate warnings and steps

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

Commit:
removing diff headers

Commit:
removing 'A:' for Answer

Commit:
fixes

haroldwongms committed:
Modify images per review notes

Renamed with hyphens
Red border

haroldwongms committed:
Updated per review notes and renamed file

haroldwongms committed:
Updated content per review notes

haroldwongms committed:
Change case in TOC for first word to be capital

GitHubber17 committed:
revised per author feedback

GitHubber17 committed:
revised per author feedback

v-nagta committed:
USER STORY 1106209 SaaS App Tutorial: Skillport Update

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
JL feedback

itechedit committed:
Update active-directory-b2c-ui-customization-custom-dynamic.md

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
typo

itechedit committed:
Update active-directory-b2c-ui-customization-custom-dynamic.md

itechedit committed:
Update active-directory-b2c-ui-customization-custom-dynamic.md

v-kents committed:
linking serverless on functions

v-kents committed:
trying just a straight a link

v-kents committed:
removed locale

mmacy committed:
Merge MicrosoftDocs/master into origin/master

Commit:
reverting tabs

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edit

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
typos

HeidiSteen committed:
typo

haroldwongms committed:
case and hyphen per review request

v-nagta committed:
updated

Commit:
branding and ruby on index

Commit:
adding docker to landing page

v-nagta committed:
updated

Commit:
rebased landing page, remove localization

Commit:
update docker logo path

Commit:
remove ruby logo from app service, update path to generic logo path

mmacy committed:
[acr] geo-rep tutorial updates

mmacy committed:
Merge branch 'master' of https://github.com/mmacy/azure-docs-pr

vhorne committed:
update for PS 1.2.11

Commit:
updated file and TOC

Commit:
title change

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
update metadata

twooley committed:
new topic for 1710 update plus TOC changes

twooley committed:
new topic for 1710 update plus TOC changes

mmacy committed:
[aks] udpate AKS CLI reference links

Commit:
dhanyah09

Commit:
dhanyah10

Commit:
remove preview feature registration requirement

Commit:
remove preview registration requirement

Commit:
update devkit to v1.2.0 release

Commit:
Update iot-hub-arduino-iot-devkit-az3166-get-started.md

Commit:
Merge pull request #26087 from cbrooksmsft/patch-4

remove preview feature registration requirement

Commit:
Merge pull request #26090 from cbrooksmsft/patch-5

remove preview registration requirement

Commit:
Merge pull request #26086 from MarkusVi/dhanyah10

dhanyah10

twooley committed:
new topic for 1710 update plus TOC changes

twooley committed:
new topic for 1710 update plus TOC changes

Commit:
Merge pull request #25922 from cynthn/faqs

FAQ fixes

brapel committed:
custom-search: fix toc

ktoliver committed:
Merge pull request #26091 from shizn/master

update devkit to v1.2.0 release

twooley committed:
new topic for 1710 update plus TOC changes

tamram committed:
Update storage-blobs-container-calculate-size-powershell.md

Fixing date and author fields.

tingqin2017 committed:
Update how-to-use-run-history-model-metrics.md

Delate unsupported logging APIs for dictionary and dataframe. Add a link for logging API reference

MicrosoftGuyJFlo committed:
Remove note for admin mobile number as this is no longer synced.

twooley committed:
new topic for 1710 update plus TOC changes

Commit:
removing subscription Ids

Commit:
Merge pull request #26082 from MarkusVi/dhanyah09

dhanyah09

gsacavdm committed:
Add compliance clarification

v-thepet committed:
fixing the toc

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

PRMerger17 committed:
Merge pull request #26102 from MicrosoftGuyJFlo/AdminMobile

Remove note for admin mobile number as this is no longer synced.

rmca14 committed:
Merge pull request #24765 from SyntaxC4-MSFT/cfowler-linux-branding

Branding Updates for App Service on Linux

Commit:
Adding info ou GetMetadataPropertyValue (for Iot Hub routes)

Commit:
Merge pull request #26101 from tingqin2017/patch-1

Update how-to-use-run-history-model-metrics.md

ktoliver committed:
Merge pull request #26020 from v-kents/1105360

linking serverless on azure functions

JasonWHowell committed:
Update howto-alert-on-metric.md

JasonWHowell committed:
Update howto-alert-on-metric.md

JasonWHowell committed:
Update howto-alert-on-metric.md

mimig1 committed:
Cosmos DB: Update for multiple funtions on same feed

mimig1 committed:
typo fix

tynevi committed:
Merge pull request #23548 from identityMy/patch-2

Update TOC.md

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gwsub

ktoliver committed:
Merge pull request #25758 from haroldwongms/openshiftupdate

Openshiftupdate

Lisaco88 committed:
Merge pull request #26080 from mmacy/aks-cli-ref-links

[aks] udpate AKS CLI reference links

PRMerger20 committed:
Merge pull request #26103 from gsacavdm/patch-3

Add compliance clarification

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gwsub

JasonWHowell committed:
Update concepts-monitoring.md

JasonWHowell committed:
Update concepts-monitoring.md

Lisaco88 committed:
Merge pull request #26105 from v-thepet/fixy

production bug: fixing the toc

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gwsub

gsacavdm committed:
Add extensions link to gov-publish

twooley committed:
new topic for 1710 update plus TOC changes

PRMerger14 committed:
Merge pull request #25874 from ggailey777/event-grid

Remove the storage gate warnings and steps

Lisaco88 committed:
Merge pull request #26107 from JasonWHowell/patch-1

MySQL: Update howto-alert-on-metric.md

Lisaco88 committed:
Merge pull request #26108 from JasonWHowell/patch-3

PostgreSQL: Update howto-alert-on-metric.md

twooley committed:
new topic for 1710 update plus TOC changes

Commit:
Merge pull request #26109 from mimig1/patch-148

Cosmos DB: Update for multiple funtions on same feed

PRMerger12 committed:
Merge pull request #26116 from JasonWHowell/patch-6

PostgreSQL Update concepts-monitoring.md

mumian committed:
remove file

GitHubber17 committed:
Merge pull request #24254 from HeidiSteen/heidist-master

Prototype Filter Docs -- placeholder files

tynevi committed:
Merge pull request #26098 from brapel/custom-search-fix-toc-overview

custom-search: fix toc

Lisaco88 committed:
Merge pull request #26117 from JasonWHowell/patch-5

MySQL: Update concepts-monitoring.md

PRMerger20 committed:
Merge pull request #26100 from MicrosoftDocs/tamram-patch-3

Update storage-blobs-container-calculate-size-powershell.md

Commit:
Merge pull request #26121 from mmacy/acr-tutorial-update

[acr] geo-rep tutorial updates

PRMerger12 committed:
Merge pull request #26106 from JSeb225/patch-14

Adding info ou GetMetadataPropertyValue (for Iot Hub routes)

Ja-Dunn committed:
Merge pull request #25390 from craigcaseyMSFT/v-craic1018

fixed nonblocking issues from PR#25221

Ja-Dunn committed:
Merge pull request #25974 from v-nagta/skillport

USER STORY 1106209 SaaS App Tutorial: Skillport Update

Commit:
Merge pull request #26118 from gsacavdm/patch-3

Add extensions link to gov-publish

Ja-Dunn committed:
Merge pull request #26124 from mumian/domainjoinedremovefile1024

remove file

ErikjeMS committed:
1102336

Ja-Dunn committed:
Merge pull request #26078 from twooley/1710relnotes

new topic for 1710 update plus TOC changes

Commit:
whatsnew04

Commit:
whatsnew04

Commit:
Merge pull request #26128 from ErikjeMS/master

1102336 per Gary

ErikjeMS committed:
1107126

JasonWHowell committed:
Update concepts-monitoring.md

JasonWHowell committed:
Update concepts-monitoring.md

ShawnJackson committed:
Merge pull request #25566 from itechedit/active-directory-b2c-ui-customization-custom-dynamic

edit pass: active-directory-b2c-ui-customization-custom-dynamic

Commit:
Merge pull request #26133 from ErikjeMS/master

1107126

Ja-Dunn committed:
Merge pull request #26069 from vhorne/upd-ps-qs

update for PS 1.2.11

ErikjeMS committed:
1084110

PRMerger17 committed:
Merge pull request #26135 from JasonWHowell/patch-8

Update concepts-monitoring.md

PRMerger16 committed:
Merge pull request #26134 from JasonWHowell/patch-7

Update concepts-monitoring.md

twooley committed:
few updates to article

Ja-Dunn committed:
Merge pull request #26113 from cherylmc/gwsub

Gwsub

PRMerger12 committed:
Merge pull request #26131 from MarkusVi/whatsnew04

whatsnew04

Commit:
fix spelling

twooley committed:
few updates to article

Commit:
Merge pull request #26141 from ErikjeMS/master

1084110

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ShawnJackson committed:
Merge pull request #25365 from GitHubber17/TFS72071

edit pass: load balancer articles (replaces 24963)

PRMerger12 committed:
Merge pull request #26147 from twooley/1710fixes

1710fixes

v-alje committed:
Merge pull request #26148 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #26146 from SergeyKanzhelev/sergkanz/fixContect

fix spelling
